weasel/_util/general.py

```diff
@@ -0,0 +1,568 @@
+import hashlib
+import logging
+import os
+import shlex
+import shutil
+import subprocess
+import sys
+import tempfile
+import warnings
+from contextlib import contextmanager
+from pathlib import Path
+from typing import (
```

Comment:
This could be reformatted as two import statements on separate lines to prevent `black` from putting each entry on its own line.